### DISCUSSION TITLE
Showcase guideline discussions

### DISCUSSION THREAD

**Discussion by katomm** [cardano-foundation/developer-portal#176] [General]
Hello,

I would like us to agree on the fact that the project showcase should be a place that someone new to the ecosystem/non-technical can come and see what can be done on Cardano, today - it is not a database to drop and promote every project. It probably doesn't make sense to make a new showcase entry for every native token.

Driven not only by discussions in #167 this thread can be used to discuss how we want to deal with showcase guidelines and how we want to evolve them. If you have any comments, suggestions, love or hate regarding the current guidelines, please discuss them here.

After reading through #167 you can find the current guidelines in the /* comments */ of https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js

**Comment by tweakch** [comment]
Important thread! I think the outcomes of the discussions here should end up in our CONTRIBUTION and (maybe) CODE OF CONDUCT... I created two PR drafts for this #183 and #184

**Comment by tweakch** [comment]
As to your point:
>  it is not a database to drop and promote every project. It probably doesn't make sense to make a new showcase entry for every native token.

I agree, there will be hundreds of NT and other projects fighting for attention. And we will be up in arms fighting to keep the showcase nice and tidy. 

While I think its good to showcase many different projects for non technical people, this is the **developer portal** and I as a dev want to see ***how things were built***... Maybe if we add a requirent to the showcase to also provide 
* github repos, 
* link white papers and 
* talk about technical details of the project in a seperate article 

...we could mitigate the potential flood of projects. 

Just raising the barrier a tiny bit can help a lot IMHO

**Reply by tweakch** [reply]
I'm just brainstorming here. how about

* feature projects through stake
* add *flair* if they "sign" their showcase through transaction metadata on mainnet
* add BAT-like `tip` feature if they add a walletaddress to their showcase
* reward projects for "minting" their git commit history on the mainnet
* Add TTL to showcases that have to be renewed by submitting PRs on a regular basis
*  put *expired* projects into `hibernation` on seperate page

Obviously the community should discuss and then vote on all of these and we should announce discussion and voting on the showcase page

**Comment by katomm** [comment]
## To recap, current guidelines for the showcase section are as follows: 

 * It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but nothing for this showcase section.
 * It has to run on Cardano mainnet.
 * It has to have a running product. (no presale, no protected pages, no coming soon messages)
 * It has to have enough community reputation.
 * It has to provide a unique value from existing showcase items. (we can't list thousands of NFT or native tokens with the current UI)
 * It has to have a stable domain name. (a random Netlify/Vercel domain is not allowed, no URL shortener, no app store links, or similar)
 * The GitHub account that adds the project must not be new. 
 * The GitHub account must have a history/or already be known in the Cardano community.
 
## I propose to add this one based on the latest discussions:
* Describe what makes your project special, avoid phrases like "the first this and that". Granular details like which project was first is tribal attribute known to cause rift and conflicts.

**Reply by rdlrt** [reply]
I'd  like to see this point added

**Comment by Digital365Staking** [comment]
I completely agree with the principle that the project showcase should highlight what’s possible on Cardano — not just list every project or token. It’s essentially the ecosystem’s “front window,” and keeping it curated makes it more useful for newcomers and more representative of Cardano’s maturity.

A few thoughts on how we might evolve the guidelines:

**- Define Clear Inclusion Criteria**

We could establish a small set of “must-have” attributes for showcase entries — for example:

The project is live and functional (not just in concept or testnet).

It demonstrates a clear use case enabled by Cardano (e.g., NFTs, DeFi, identity, infrastructure).

It has public-facing resources (website, repo, or product).
This would help maintain a balance between inclusiveness and relevance.

**- Categorization over Volume**

Instead of accepting every submission, we could categorize projects under a few thematic sections like:

Applications (wallets, marketplaces, DeFi apps)

Infrastructure & Tools (SDKs, libraries, explorers)

Community & Education (initiatives, learning platforms)
That way, visitors can explore the ecosystem meaningfully without being overwhelmed.

**- Handling Native Tokens**

I agree — listing each native token as a separate showcase item would quickly dilute the value of the section. If we still want to acknowledge the token ecosystem, maybe we could feature a curated “Token Ecosystem Overview” page summarizing notable examples, rather than adding individual showcase entries.

**- Submission Review Process**

It might help to define a light review workflow for new entries — e.g., a short checklist or tag ("showcase: pending review") to ensure consistency before merging PRs. This would prevent guideline drift over time.

**- Future Vision**

Long term, the showcase could evolve into a dynamic gallery that connects to ecosystem metrics (e.g., GitHub activity, on-chain stats, or Catalyst funding status) — giving visitors both inspiration and data-backed insight into Cardano’s activity.

Overall, I think focusing on quality and representativeness rather than quantity will help maintain the Developer Portal’s credibility and make it much more valuable for new users.
